package.json

```diff
@@ -225,15 +225,15 @@
     "typescript-formatter": "^7.2.2",
     "vscode-extension-tester": "^3.0.1",
     "vscode-test": "^1.3.0",
-    "webpack": "^4.42.1",
-    "webpack-cli": "^3.3.11"
+    "webpack": "^5.50.0",
+    "webpack-cli": "^4.7.2"
```

Comment:
Didn't we have some issues with upgrading webpack before? Can we double check that we're not going to hit those issues again?

Reply:
We did, yes. I did a number of checks on this specifically, blowing away `node_modules` and starting over a number of times, and couldn't reproduce whatever issue we had (I can't remember what the issue was). This looks like it works, however.